internal/utils/ci_providers.go

```diff
@@ -110,10 +132,14 @@ func extractAppveyor() map[string]string {
 	tags[constants.CIProviderName] = "appveyor"
 	if os.Getenv("APPVEYOR_REPO_PROVIDER") == "github" {
 		tags[constants.GitRepositoryURL] = fmt.Sprintf("https://github.com/%s.git", os.Getenv("APPVEYOR_REPO_NAME"))
-		tags[constants.GitCommitSHA] = os.Getenv("APPVEYOR_REPO_COMMIT")
-		tags[constants.GitBranch] = firstEnv("APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH", "APPVEYOR_REPO_BRANCH")
-		tags[constants.GitTag] = os.Getenv("APPVEYOR_REPO_TAG_NAME")
+	} else {
+		tags[constants.GitRepositoryURL] = os.Getenv("APPVEYOR_REPO_NAME")
```

Comment:
does this make sense? when it's not github it's always a URL?

Reply:
good point, this is based on the current implementation on other languages, we are handling only for github.

According to: https://www.appveyor.com/docs/environment-variables/

Appveyor support the following providers:

```
APPVEYOR_REPO_PROVIDER - gitHub, bitBucket, kiln, vso, gitLab, gitHubEnterprise, gitLabEnterprise, stash, gitea, git, mercurial or subversion
```

Maybe we should also add the support bitBucket and gitLab. But we should do that as a global issue for all tracers, so I'll leave that to another PR.